infrastructure/github-env-setup.yml

```diff
@@ -1,3 +1,5 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
question: is this actually needed? it was working before even without it.
Just my curiosity

Reply:
it's useful to let syntax analysis and completion support in IDEs such as JetBrains family with AWS plugin